index.html

```diff
@@ -5663,7 +5663,21 @@ <h3>Language and Base Direction</h3>
   "@id": "https://www.w3.org/2018/credentials/examples#title"
 }
         </pre>
+      </section>
+
+      <section class="informative">
+        <h3>Providing Default Language and Direction</h3>
 
+        <p>
+A default language and direction MAY be provided for the entire credential.
+However, doing so introduces a higher demand on creating a compliant processor
+and fully compliant context files. Consequently, using default language and
+direction should be limited to scenarios where JSON-LD expansion-based
+transformation can be assured. See the
+<a href="https://www.w3.org/TR/json-ld11/#string-internationalization">String
+Internationalization</a> section of the [[JSON-LD]] specification for more
+information.
```

Comment:
Hmm... we're almost back where we started.

The `name` and `description` fields really must have a language and base direction and consumers of credentials need to be able to find out what it is. Either the document has a default for these or it uses the language value structure. "Neither" should not be an option, but that is what this MAY (combined with the rest of the document) results in--particularly since this paragraph makes it sound like a bad idea to provide the default.

Reply:
Perhaps we could do a "MUST either provide human-readable fields via the language value structure (preferred) or via document defaults"?

Reply:
`SHOULD provide language and base direction for human-readable fields either via the language value structure (preferred) or via document defaults` because the MUST horse has [long since left the barn](https://github.com/w3c/vc-data-model/pull/1339#discussion_r1391390907).